lib/main.coffee

```diff
@@ -358,3 +360,25 @@ module.exports = Hydrogen =
                 @onKernelChanged kernel
 
         @wsKernelPicker.toggle @editor.getGrammar()
+
+
+    copyPathToConnectionFile: ->
+        grammar = @editor.getGrammar()
+        language = @kernelManager.getLanguageFor grammar
+
+        unless @kernel?
+            message = "No running kernel for language `#{language}` found"
```

Comment:
I think the message should be `No running kernel associated with this file` to account for the case  a kernel hasn't been launched yet.

This would make lines 366 and 367 unnecessary.